.golangci.yml

```
@@ -1,128 +1,488 @@
+        linters: [ revive, staticcheck ]
+      - path: '_test\.go'
+        linters:
+          - bodyclose
+          - dupl
+          - errcheck
+          - funlen
+          - goconst
+          - gosec
+          - noctx
+          - wrapcheck
+          - exhaustruct
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)